src/components/charts/ReferencesTable.tsx

```diff
@@ -18,6 +22,7 @@ export default function ReferencesTable() {
   const [pageLength, setPageLength] = useState(5);
   const [column, setColumn] = useState('denominator');
   const [direction, setDirection] = useState('descending');
+  const [searchResults, setSearchResults] = useState<AirtableRecord[] | undefined>(undefined);

```

Comment:
Couldn't the initial value for searchResults just be an empty array instead of undefined? That would simplify typing

Reply:
There is a slight difference between an empty array of records vs undefined. Undefined means that no search has been made, the search query is empty. An empty array of records means that a search was made but no results were found. We show the full set of airtable records by default when the searchResult is undefined, and nothing when the records array is empty